### ISSUE TITLE
Document what the mesmer sample data files are

### ISSUE THREAD

**Issue by dchaley** [dchaley/deepcell-imaging#25]
Update file: sample-data/deepcell/README.md

**Comment by lynnlangit** [comment]
Is this complete?

**Comment by dchaley** [comment]
No. The TODOs in referenced file linking back to this issue need to be addressed: [sample-data/deepcell/README.md](https://github.com/dchaley/deepcell-imaging/tree/main/sample-data/deepcell)
